seedfile_shift.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
Yeah it should be in helpers folder. But atm the script needs to be in PATH for the client to work. I will rebuild the functionality in Fortran instead and add the code to the client. We can still leave this script in helpers then.